server/index.js

```diff
@@ -0,0 +1,5 @@
+const express = require('express');
+const app = express();
+const port = 3002;
+
+app.listen(port, () => {console.log(`Listening on port ${port}`)});
```

Comment:
@MarissaShieh Please add a newline break at the end of the file to meet style guide.